### ISSUE TITLE
Modification: Requiring notaries to self-circumvent on projects they engage in as SPs

### ISSUE THREAD

**Issue by Carohere** [filecoin-project/notary-governance#602]
## Issue Description
The recent behavior of some notaries in granting DC has caused confusion and a lack of consensus as to whether they are properly exercising their authority as notaries. Some notaries who hold dual notary+sp status are constantly issuing DCs for applications that share sp partnership within the relevant project. Although the DC disclosure is made in advance, it is still not convincing. However, since there are no explicit rules, it is prone to different perceptions of such behavior, so a uniform standard is needed to avoid confusion and any potential collusion.

## Impact
Greatly enhance the credibility of the application and the transparency of the community

## Proposed Solution(s)
Prohibit notaries from issuing any application with potential self-interest. It requires self-regulation by the notary, as well as oversight through community members. 

- If any irregularities are discovered/highly suspected, the DC release should be halted during the investigation and all the powers held by the notary need to be suspended. 

- If the violation is proven, the application performed shall be suspended (either one or more) and penalties shall be imposed on the notary. Disqualifying the V3 notary for the current round and a 0.2-point deduction for each violation if the notary participates in the next round of election.

### Timeline
ASAP - can be implemented right away.

### Technical dependencies 
Since it is impossible to predict new projects and each notary has different connections, I don't know if this is technically possible, and certainly better if it is applicable.

### End of POC checkpoint (if applicable)
N/A

## Risks and mitigations 
Risk-free.

## Related Issues
From what I've seen, this is a somewhat common occurrence. Everyone is more than welcome to point out applications that they personally find a bit odd in terms of self-involvement.

**Comment by Carohere** [comment]
Hi @Dominic92, I saw that you made proposals on slack regarding notaries with multi-identities. I've contacted you via DM to ask if I could quote your idea, haven't heard you back so I didn't include it here and I wonder if you would like to share more about it? 
https://filecoinproject.slack.com/archives/C01DLAPKDGX/p1660618053070089

**Comment by Kevin-FF-USA** [comment]
![image](https://user-images.githubusercontent.com/94004586/186200144-ea217cf0-bcb4-4394-99ec-8a3d1903cef6.png)

**Comment by dannyob** [comment]
I'm a fan of making a rule first that notaries should disclose: I think that's our presumption now, and it would give us a better idea of what the relationships are. It's also far easier to police disclosure than it is to write rules about relationships.

**Comment by kernelogic** [comment]
Just writing down what I said during the zoom meeting: I agree client disclosure is ok, and actually is a good thing to do, making due diligence easier. 

However prohibiting notaries to accept any deals from the LDN they signed, will discourage their participation. Almost all notaries are SPs as well, and they are the most active members in the community.

**Comment by psh0691** [comment]
The Fileplus project is a tremendous help to the growth of the Filecoin network. But there are many difficult problems.
In the plus diagram, the notary is associated with the client. The client is connected to the SP. Therefore, there is no connection between the notary and the SP.
However, most pure clients are ignorant of the Filecoin network. They don't know about GitHub, Slack, or even the Fileplus project. Currently, the SP invites and participates with clients. 

How about the notary? Most are SP or associated. Notaries verify clients and their data and allocate DCs, but it would be reasonable to view them as service organizations rather than audit agencies or approval agencies.

What I want to say is that although it is not listed in the Plus Operating Guidelines, it is active support for the SP of PL, Plus Team, and Notary Public. SPs are the main pillars of the Filecoin network and play the biggest role in ecosystem expansion and growth. It should be remembered that the most important thing to support is SP rather than clients or notaries. 

In conclusion, I hope that the notary public's DC approval activities will be organized so that SP's activities are not restricted as much as possible.

**Comment by Carohere** [comment]
Hi @dannyob, I may not have been very clear at the last meeting. Sorry for causing confusion.  I'm not against disclosure and it is not incompatible with my proposal. In fact it would even be better to achieve a high level of transparency. If notaries made complete disclosures, yes this could theoretically stop all self-dealing, avoid collusion, and prevent DC abuse. **But policing disclosures presuppose that notaries are willing to do so. We have no way of knowing whether they are making full or partial disclosures that are simply uncontrollable.** 

@kerenlogic Fei, happy to know that we' re on the same page about client disclosure. Yes, almost all notaries are SPs. The core of this proposal is not to exclude notaries from being active as sp, not at all. **What it requires is merely to act in a single role in the same application/project in order to avoid any suspicion about the credibility of the notary and the authenticity of the application.**

I don't think it is difficult to choose between roles as sp or notary. These two do not pose as a dilemma. It is natural to choose to participate as a sp because of the priority of interests. And **if the application is genuine there is no need to worry about whether it will be supported by other notaries.** I mean there are plenty active notaries left.

**Comment by Carohere** [comment]
Hi @psh0691, thanks for joining us.

1. Yes. As per diagram, there should not be any direct connection between notary and sp. I don't know if blockchainworld is directly involved in mining or not. From what I saw in the notary application, it seems like your organization is only working on blockchain-related media. The whole point is that only a very small number of notaries act as a single role like you( if i'm right on that). Most of them are involved as SPs trying to bring more storage capacity into filecoin which is awesome. But as long as they take two roles in one application, the direct link of interest would most certainly cause potential opacity and deals happen under the table. This is inconsistent with transparency which we believe in this community. It is and will be damaging the future of feilcoin.

2. There're 50+ notaries, i don't see the need to worry genuine applications won't get enough support. TBH I don't even think one or two or even five notaries' absence will stop any legit application moving forward. 

BTW thanks a lot for the timely and comprehensive news. 항상 잘 보고 있습니다~

**Comment by Carohere** [comment]
I think a lot people have misundertood my proposal. **Once again, I want to clarify that this proposal is NOT meant to restrict activities of sps.** It makes perfect sense that they would choose to participate in the project as SPs since notaries have no income from a monetary perspective. But taking two roles in one application is dubious and it shouldn't happen.

**Comment by Carohere** [comment]
Recapping the points included in this proposal so far. 

**Pros**
- Limit self-dealing, collusion to a certain extent
- Reduce disputes and save the time of community members 
- Enhance community transparency
- Improve credibility of the next round notary candidates

**Cons**

- Cannot completely prevent collusion and potential self-dealing (as there is no way to verify whether the notary nodes are fully disclosed or not)
- The client may not be able to meet the expected progress with their app (which can be achieved with the support of other notaries)

**About governance team's response from the last GC** 

@dkkapur @raghavrmadya I disagree that this modification would limit applications, actually quite the opposite. From those apps below, you can clearly see that the client/notary applying on behalf of the client has disclosed the SPs they will be working with. Self-circumvention of SP will simplify due diligence to a certain extent as there is no need to worry about direct collusion. I believe this actually will improve the overall efficiency of application process. 
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/420
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/425
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/488

Lastly, I would like to mention a VERY important point. There are no measures to bring transparency to perfection immediately. I know this proposal isn't perfect but at least it's better to do something imperfectly than to do nothing perfectly.

**Comment by Chris00618** [comment]
There are lots of proposals recently. Why can't we use voting tool to make quick decisions and let the REAL clients participate in making rules related to themselves? This can reflect the spirit of web3.
